packages/loader/container-loader/src/collabWindowTracker.ts

```diff
@@ -69,9 +69,14 @@ export class CollabWindowTracker {
         }
 
         this.opsCountSinceNoop++;
-        if (this.opsCountSinceNoop >= this.NoopCountFrequency) {
-            this.submitNoop(false /* immediate */);
-            return;
+        if (this.opsCountSinceNoop === this.NoopCountFrequency) {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
+            Promise.resolve().then(() => {
```

Comment:
It would be great to add test coverage to ensure we are actually sending 1 op here. I.e. test would send 1K ops in a row and validate that this logic will generate one noop in resposne. This should be possible to do in UT, i.e. without having Container or DeltaManager objects around

Reply:
i think the ones in the deltaManager unit tests are doing this already